.github/workflows/gradle-run-unit-tests.yml

```diff
@@ -64,19 +64,21 @@ jobs:
         uses: actions/upload-artifact@v4
         with:
           name: report
-          path: app/build/reports/jacoco
+          path: app/build/reports/kover
 
       - name: Download Test Reports Folder
         uses: actions/download-artifact@v4
         with:
           name: report
-          path: app/build/reports/jacoco
+          path: app/build/reports/kover
           merge-multiple: true
 
-      - name: Upload Test Report
+      - name: Upload code coverage to codecov
         uses: codecov/codecov-action@v4
+        env:
+            CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
Adjustment, due to breaking changes from: 
- https://github.com/wireapp/wire-android/pull/2663
- https://github.com/codecov/codecov-action?tab=readme-ov-file#breaking-changes